### ISSUE TITLE
Deployable Environment

### ISSUE THREAD

**Issue by elstob** [mtgred/netrunner#51]
I'd love to help contribute to this project but the installation / running instructions are sadly insufficient and result in multiple errors trying to install on a fresh Ubuntu instance. More/better instructions would be great but ideally a deployable Vagrant box (or similar) that provisions itself would be put in place.

This might already be on your list of things to do, but if not I definitely recommend it going forward. I'm happy to help with getting this project to an easily deployable state as well but step 1 of that is getting it running at all :smile: 

Thanks and great work so far!

**Comment by mtgred** [comment]
Hi, what errors are you getting? It will help me identify what's lacking in the setup instructions.

**Comment by elstob** [comment]
I was able to get a running server in the end but the issues I found along the way included:

- Compilation issues with non-stable (i.e. recent) versions of Node as well as the standard apt Node package. Recommend suggesting users install 0.10.35 via nvm.
- apt Leiningen version is 1.X something not 2+ which causes failures all round as you'd imagine.
- bower isn't installed at any point but nor is listed as a prerequisite.

Other minor issues:

- Suggest file compilation before attempting to run server.coffee (otherwise it just fails with a .game not found error).
- Have the data fetch script finish instead of hanging/waiting indefinitely.

Now that I've been able to go through the steps to get a server running I can take a look at automating that and creating a Vagrant box that would allow people to spin up their own dev server VM quickly.

Thanks!
